app/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/meldekort/domene/MeldekortBehandlinger.kt

```diff
@@ -33,6 +32,36 @@ data class MeldekortBehandlinger(
 
     val log = KotlinLogging.logger { }
 
+    val periode: Periode by lazy { Periode(verdi.first().fraOgMed, verdi.last().tilOgMed) }
+    val sakId: SakId by lazy { verdi.first().sakId }
+
+    private val behandledeMeldekort: List<MeldekortBehandlet> by lazy { verdi.filterIsInstance<MeldekortBehandlet>() }
+    private val sisteMeldekortPerKjede: List<MeldekortBehandling> by lazy { verdi.groupBy { it.kjedeId }.values.map { it.last() } }
+    val sisteBehandledeMeldekortPerKjede: List<MeldekortBehandlet> by lazy { sisteMeldekortPerKjede.filterIsInstance<MeldekortBehandlet>() }
+
+    /** Under behandling er ikke-avsluttede meldekortbehandlinger som ikke er til beslutning. */
```

Comment:
Kan det bare finnes et meldekort som er under behandling av gangen?

Reply:
Gikk litt fram og tilbake på om vi kunne tillate flere åpne behandlinger, men landet på at var tryggest å beholde restriksjonen på en åpen behandling. Ettersom korrigering tilbake i tid kan påvirke utbetaling av påfølgende meldeperioder, så kan det oppstå inkonsistente beregninger ved flere samtidige behandlinger.

Tenker det ikke er noe stort behov å kunne ha flere åpne behandlinger, så foretrekker å ikke innføre mer kompleksitet for å holde samtidige behandlinger i sync. 😅